portable/ThirdParty/GCC/Posix/port.c

```diff
@@ -141,7 +144,7 @@ portSTACK_TYPE * pxPortInitialiseStack( portSTACK_TYPE * pxTopOfStack,
      */
     thread = ( Thread_t * ) ( pxTopOfStack + 1 ) - 1;
     pxTopOfStack = ( portSTACK_TYPE * ) thread - 1;
-    ulStackSize = ( pxTopOfStack + 1 - pxEndOfStack ) * sizeof( *pxTopOfStack );
+    ulStackSize = (size_t)( pxTopOfStack + 1 - pxEndOfStack ) * sizeof( *pxTopOfStack );
```

Comment:
```suggestion
    ulStackSize = ( size_t )( pxTopOfStack + 1 - pxEndOfStack ) * sizeof( *pxTopOfStack );
```

Reply:
Changed, but if you require this level of formatting compliance, please incorporate it into `uncrustify` definition and/or use `clang-format` which has this level of formatting definition